application/wrapper/storage_mock.go

```diff
@@ -0,0 +1,138 @@
+package wrapper
```

Comment:
Should use [mockgen](https://github.com/golang/mock) to generate these mocks instead of doing by hand.